client/src/test/java/com/sinch/sdk/domains/voice/adapters/CalloutsServiceTest.java

```diff
@@ -67,8 +58,20 @@ void callTTS() throws ApiException {
   @Test
   void callCustom() throws ApiException {
```

Comment:
Nit: to be renamed `custom()` to be aligned with the name of the tested method as it has been done for `conference()` and `textToSpeech()`

Reply:
Done